src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml

```diff
@@ -142,7 +142,7 @@
 				Value="Compact" />
 		<Setter Property="ExitDisplayModeOnAccessKeyInvoked"
 				Value="False" />
-		<Setter Property="CornerRadius"
+		<not_win:Setter Property="CornerRadius"
```

Comment:
This mean we will note have CornerRadius for WinUI @kazo0 ?

Reply:
Honestly I don't know right now. It seems like you can't reference the ControlCornerRadius resource when running a WinUI app

Reply:
I've apparently done this before and I have no memory of this: https://github.com/unoplatform/Uno.Themes/pull/586

Not sure if maybe @jeromelaban would know why we needed to put either `not_win:` or `contract7NotPresent:`